protocols/imap/src/main/java/org/apache/james/imap/processor/AbstractMailboxProcessor.java

```diff
@@ -332,6 +332,11 @@ protected void okComplete(ImapRequest request,  ResponseCode code, ImapProcessor
         responder.respond(response);
     }
 
+    protected void okSuccess(ImapRequest request, ImapProcessor.Responder responder) {
+        final StatusResponse response = factory.taggedOk(request.getTag(), request.getCommand(), HumanReadableText.SUCCEEDED);
```

Comment:
```suggestion
        StatusResponse response = factory.taggedOk(request.getTag(), request.getCommand(), HumanReadableText.SUCCEEDED);
```

Avoid final qualifier on variables.